### ISSUE TITLE
HTML title during enumeration

### ISSUE THREAD

**Issue by alfonsocaponi** [maurosoria/dirsearch#764]
It would be very useful collect and report the HTML title during entrypoint enumeration.
I don't know if easy to make...
What do you think? :)

**Comment by shelld3v** [comment]
Hi, this is super easy to make, but the problem is it will make the output looks dirty

**Comment by alfonsocaponi** [comment]
True, but if it were **optional**... :)
For example, putting title in square brackets. Just a note, remove "multi spaces" (keep only single spaces) and "carriage return":

200 -   17KB - https://example.com:443/docs/ **[Apache Tomcat 8 (8.5.29) - Documentation Index]**

In my opinion it would be an added value during bug bounty activities! :)

**Comment by shelld3v** [comment]
Browsing an URL is one of the most basic skills in bug bounty hunting! You can't do it;)

**Comment by shelld3v** [comment]
>True, but if it were optional... :)

That fact that we can make tons of optional flags, but it will just flood the dirsearch flags. We only focus on important things, and remember that even you see the title, browsing that endpoint is still a must-do action

**Comment by alfonsocaponi** [comment]
Yes, I agree with you. It would just be a way to filter through large numbers :)

**Comment by shelld3v** [comment]
> Yes, I agree with you. It would just be a way to filter through large numbers :)

Yooo!

I am working on a PR, that allows you to create your own **signatures**. With a signature (YAML format), any response that matches the regex/text in the signature will be notified in the output. Is this a good replacement for your suggestion?
